src/main/java/com/sonyericsson/jenkins/plugins/bfa/model/indication/FoundIndication.java

```diff
@@ -45,10 +44,6 @@
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class FoundIndication {
 
-    /**
-     * The platform file encoding. We assume that Jenkins uses it when writing the logs.
-     */
-    protected static final String FILE_ENCODING = System.getProperty("file.encoding");
```

Comment:
could be in use by externally implemented extension point (because `protected`) So better to keep but `@Deprecate`.

Reply:
what could be done by it though?
This does nothing.

I've searched the jenkinsci organisation and the only external plugin using this class is the claim-plugin and its not using this
https://github.com/search?q=org%3Ajenkinsci+com.sonyericsson.jenkins.plugins.bfa.model.indication.FoundIndication&type=Code

Reply:
Change reverted anyway, it was unrelated